tests/subsys/nrfs/sysbuild.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+  message(FATAL_ERROR "REMOTE_BOARD must be set to a valid board name")
+endif()
+
+# Add remote project
+ExternalZephyrProject_Add(
+    APPLICATION remote
+    SOURCE_DIR ${APP_DIR}/remote
+    BOARD ${SB_CONFIG_REMOTE_BOARD}
+    BOARD_REVISION ${BOARD_REVISION}
+  )
+
+# Add a dependency so that the remote image will be built and flashed first
+add_dependencies(nrfs remote)
+# Add dependency so that the remote image is flashed first.
+sysbuild_add_dependencies(FLASH nrfs remote)
```

Comment:
The second comment could be dropped as the first one already explains everything.